samples/build.gradle.kts

```diff
@@ -0,0 +1,27 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I think it's unclear what this sample is trying to demonstrate:
- It doesn't clearly indicate it's for testing local GeckoViews (though the gradle-plugin readme mentions that use case)
- It doesn't demonstrate how to apply the org.mozilla.android gradle-plugin from plugins.mozilla.org so isn't really applicable to external consumers.

This seems like less of a sample for external consumers and more like a way to test against `gradle-plugin` locally (maybe this is my fault for saying we needed a "sample app" in the readme, though maybe there are benefits to similar naming to a-c). If so, should we put this in `./gradle-plugin/testProject/` or something instead?

In any case, it also feels weird that this is at the root of the repository given that this repo could, in theory, have many other things in it but I can see reasons for keeping it at the top.